.github/workflows/master.yml

```diff
@@ -27,7 +27,7 @@ env:
 jobs:
   lint:
     runs-on: ubuntu-latest
-    timeout-minutes: 45
+    timeout-minutes: 30
```

Comment:
45 is too long. 30 should be enough.

Reply:
This job takes:

- 10~12 minutes when cache is available.
- 5~7 minutes when cache is not available.